app/src/lib/git/core.ts

```diff
@@ -255,6 +255,8 @@ function getDescriptionForError(error: DugiteError): string {
       return 'This path is not a valid path inside the repository.'
     case DugiteError.LockFileAlreadyExists:
       return 'A lock file already exists in the repository, which blocks this operation from completing.'
+    case DugiteError.NoMergeToAbort:
+      return 'There is no merge operation to abort.'
```

Comment:
maybe something like "There's no merge in-progress, so we there's nothing to cancel."

What do people think about the use of **cancel** merge over **abort** merge?

Reply:
cc @desktop/design